docs/data-cookbooks/browse-generation.md

```diff
@@ -79,20 +79,20 @@ For this example, you are going to be adding two workflows to your Cumulus deplo
 
 * DiscoverGranulesBrowseExample
 
-  This workflow will run the ```DiscoverGranules``` step, targeting the S3 bucket/folder mentioned in the prerequisites.    The output of that step will be passed into QueueGranules, which will trigger the second workflow for each granule to be ingested.   The example presented here will be a single granule with a .hdf data file and a .met metadata file only, however your setup may result in more granules, or different files.
+  This workflow will run the ```DiscoverGranules``` task, targeting the S3 bucket/folder mentioned in the prerequisites.    The output of that task will be passed into QueueGranules, which will trigger the second workflow for each granule to be ingested.   The example presented here will be a single granule with a .hdf data file and a .met metadata file only, however your setup may result in more granules, or different files.
 
 
 * CookbookBrowseExample
 
-  This workflow will be triggered for each granule in the previous workflow.    It will utilize the SyncGranule lambda, which brings the files into a staging location in the Cumulus buckets.
+  This workflow will be triggered for each granule in the previous workflow.    It will utilize the SyncGranule task, which brings the files into a staging location in the Cumulus buckets.
 
-  The output from this step will be passed into the 'ProcessingStep', which in this example will utilize a 'fake' processing lambda we provide for testing/as an example in Core, however to use your own data you will need to write a lambda that generates the appropriate CMR metadata file and accepts and returns appropriate task inputs and outputs.
+  The output from this task will be passed into the ```ProcessingStep``` step , which in this example will utilize the ```FakeProcessingLambda``` task we provide for testing/as an example in Core, however to use your own data you will need to write a lambda that generates the appropriate CMR metadata file and accepts and returns appropriate task inputs and outputs.
```

Comment:
👍 This is good. A note about+link to `nasa/cumulus-process` as a starting point might be helpful for people looking to write process tasks.

Reply:
I think we probably ought to improve https://nasa.github.io/cumulus/docs/workflows/lambda (probably outside this PR) and leave this tutorial with the reference to "knowing basics of lambda construction/tooling", but I'm open to counter arguments

Reply:
That's fair. I'll put in a ticket with my thoughts on documenting that library in our core docs.